frameworks/helloworld/universe/package.json

```diff
@@ -2,7 +2,7 @@
   "packagingVersion": "4.0",
   "upgradesFrom": ["*"],
   "downgradesTo": ["*"],
-  "minDcosReleaseVersion": "1.10",
+  "minDcosReleaseVersion": "1.13",
```

Comment:
This is not strictly mandatory but it still makes sense to have this just to ensure we don't deploy `seccomp.yaml` in older clusters.